_data/ome2021.json

```diff
@@ -311,7 +311,8 @@
             },
             {
               "@type": "Event",
-              "name": "OMERO.figure workshop",
+              "name": "OMERO.figure Workshop",
+              "description": "This session will show how figures can quickly be created using OMERO.figure.",
```

Comment:
I would add:

"New workflows showing a more advanced work with ROIs in OMERO.figure will be presented."